stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -634,7 +634,7 @@ function(_compile_swift_files
   list(APPEND swift_flags "-enable-experimental-feature" "SE427NoInferenceOnExtension")
   list(APPEND swift_flags "-enable-experimental-feature" "AllowUnsafeAttribute")
 
-  list(APPEND swift_flags "-enable-experimental-feature" "NonescapableTypes")
+  list(APPEND swift_flags "-enable-experimental-feature" "LifetimeDependence")
```

Comment:
My expectation is that it is not safe to remove `NonescapableTypes` here until all older compilers we support have it enabled by default. Until then, `-enable-experimental-feature NonescapableTypes` needs to appear in the `.swiftinterface` so that the older compilers allow use of non-escapable types in the code in the interface.